Python/bltinmodule.c

```diff
@@ -3014,9 +3014,16 @@ static PyMethodDef builtin_methods[] = {
 };
 
 PyDoc_STRVAR(builtin_doc,
-"Built-in functions, exceptions, and other objects.\n\
+"Built-in functions, types, exceptions, and other objects.\n\
 \n\
-Noteworthy: None is the `nil' object; Ellipsis represents `...' in slices.");
+This module provides direct access to all 'built-in'\n\
+identifiers of Python; for example, `builtins.open` is\n\
+the full name for the built-in function `open()`.\n\
```

Comment:
What's the convention regarding backticks (and other markup syntax)? In my experience, most stdlib docstrings are just plain text.

Reply:
Based on a quick script, there are 11 modules in total which use backticks (including `builtins`) so definitely not standard. Not using any quotes seems pretty common so I could go with that.

Reply:
Sounds good. Thanks for checking!